packages/core/src/manifest.ts

```diff
@@ -63,7 +63,7 @@ async function getDevInstanceMetadata(
   }
 
   return {
-    networkName: 'hardhat',
+    networkName: hardhatMetadata.clientVersion.startsWith('anvil') ? 'anvil' : 'hardhat',
```

Comment:
My only concern is that this value is depending on `clientVersion` to always start with `anvil`, and may suddenly not be true again.

Nonblocking, we already chatted about this and it's not an issue. Just for the record.

Reply:
Changed this to try calling `anvil_metadata` to determine if Anvil, otherwise try `hardhat_metadata` to determine if Hardhat, otherwise conclude that it is neither.